www-plugins/passff-host/passff-host-1.0.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1
+
+DESCRIPTION="Host app for the PassFF WebExtension"
+HOMEPAGE="https://github.com/passff/passff-host"
+
+# Using raw because of difference between git and release
+# See https://github.com/passff/passff-host/issues/18
```

Comment:
I went against it because passff still haven’t got a proper version check of passff-host, so the content at the URL can still change, which could break the Manifest file.
I would have used a commit hash if there was fixes, but there is none (if I consider https://github.com/passff/passff-host/pull/3 to not be one, and exclude the shell script we aren’t using)